js/exporter/jspdf/v3/draw_utils.js

```diff
@@ -63,18 +75,31 @@ function drawBorders(doc, rect, drawLeftBorder = true, drawRightBorder = true, d
     }
 }
 
-function setCurrentFont(doc, cell, styles) {
-    const font = isDefined(cell.font) ? extend({}, styles.font, cell.font) : styles.font;
+function setTextStyles(doc, { textColor, font }, docStyles) {
+    const _textColor = isDefined(textColor) ? textColor : docStyles.textColor;
+    if(_textColor !== doc.getTextColor()) {
+        doc.setTextColor(_textColor);
+    }
+
+    const _font = isDefined(font) ? extend({}, docStyles.font, font) : docStyles.font;
```

Comment:
In general variable with "_" menas private property.
Maybe it's better to rename it to cellFont or currentFont?

Reply:
Ok, good idea. Let's do this renaming.